src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -548,22 +664,21 @@ public void onChildDraw(Canvas c, RecyclerView recyclerView,
         Paint p        = new Paint();
         float alpha    = 1.0f - Math.abs(dX) / (float) viewHolder.itemView.getWidth();
 
-        if (dX > 0) {
+        if (dX < 0) {
           Bitmap icon;
 
-          if (archive) icon = BitmapFactory.decodeResource(getResources(), R.drawable.ic_unarchive_white_36dp);
-          else         icon = BitmapFactory.decodeResource(getResources(), R.drawable.ic_archive_white_36dp);
+          icon = BitmapFactory.decodeResource(getResources(), R.drawable.ic_delete_white_24dp);
```

Comment:
why is it 24dp when the old icons were 36dp?

Reply:
I believe that was just the trash icon resource that already existed within signal.

Reply:
Weird that they have different density independent measurements. If it looks the same size in the UI I guess its fine.